code/__DEFINES/~~bubber_defines/rr_optin_defines.dm

```diff
@@ -0,0 +1,20 @@
+#define RR_OPT_IN_STRING "Yes"
+#define RR_OPT_OUT_STRING "No"
+
+/// Assoc list of stringified opt_in_## define to the front-end string to show users as a representation of the setting.
+GLOBAL_LIST_INIT(rr_opt_in_strings, list(
+	"1" = RR_OPT_IN_STRING,
+	"0" = RR_OPT_OUT_STRING,
+))
+
+/// Assoc list of stringified opt_in_## define to the color associated with it.
+GLOBAL_LIST_INIT(rr_opt_in_colors, list(
+	RR_OPT_IN_STRING = COLOR_RED,
+	RR_OPT_OUT_STRING = COLOR_EMERALD
+))
+
+/// Round removal opt-in define
+#define RR_OPT_IN TRUE
+
+/// Prefers not to round removed.
+#define RR_OPT_OUT FALSE
```

Comment:
Seems my point on these was misunderstood. What I meant was that since we no longer have several different levels to this, you remove these defines, and just use TRUE and FALSE in their place, and rename the proc names to convey that

Reply:
just change the pref to a toggle honestly